src/lib.rs

```diff
@@ -40,6 +43,24 @@ static ALLOC: self::internal_telemetry::allocations::Allocator<tikv_jemallocator
         tikv_jemallocator::Jemalloc,
     );
 
+#[cfg(any(
+    all(feature = "mimalloc", target_os = "macos"),
+    all(feature = "mimalloc", not(feature = "allocation-tracing"))
+))]
+#[global_allocator]
+static ALLOC: mimalloc::MiMalloc = mimalloc::MiMalloc;
+
+#[cfg(all(
+    feature = "mimalloc",
+    feature = "allocation-tracing",
+    not(target_os = "macos")
+))]
+#[global_allocator]
+static ALLOC: self::internal_telemetry::allocations::Allocator<mimalloc::MiMalloc> =
+    self::internal_telemetry::allocations::get_grouped_tracing_allocator(
+        mimalloc::MiMalloc,
```

Comment:
I believe we currently want to keep Jemalloc for allocation tracking here.

Reply:
Okay, I simply remove this part